docs/notebooks/batch_optimization.pct.py

```diff
@@ -87,7 +87,7 @@ def build_model(data):
 batch_ei_acq = BatchMonteCarloExpectedImprovement(sample_size=1000, jitter=1e-5)
 batch_ei_acq_rule = EfficientGlobalOptimization(  # type: ignore
     num_query_points=10, builder=batch_ei_acq)
-points_chosen_by_batch_ei, _ = batch_ei_acq_rule.acquire_single(search_space, initial_data, model)
+points_chosen_by_batch_ei = batch_ei_acq_rule.acquire_single(search_space, initial_data, model)
```

Comment:
Would be good to also have an example of using TrustRegion in one of the notebooks (maybe in ecovering_from_errors like you did before)?

Reply:
i'm lost. I haven't changed that notebook